packages/react-i18n-universal-provider/src/utilities.ts

```diff
@@ -0,0 +1,13 @@
+function pruneUndefinedFields(obj = {}) {
+  Object.keys(obj).forEach(key => obj[key] === undefined && delete obj[key]);
```

Comment:
nit: it feels a bit weird that this both mutates the object and also returns it, usually I think we'd want to either return a totally new object (especially in react contexts where object identity matters) or only mutate it and not return anything (so that it is clear that we are relying on side-effects)

It's not a big deal since this function isn't really exposed anywhere though.

Reply:
Yeah I see what you mean, I guess I did it like this because it made the destructuring in the other function a bit more concise, and also deleting undefined fields seemed like an inconsequential mutation to me. I'll change it so that it just returns a new object just for cleanliness' sake.